Resources/Prototypes/DeltaV/Entities/Objects/Devices/station_beacon.yml

```
@@ -159,3 +159,11 @@
   components:
   - type: NavMapBeacon
     text: station-beacon-prosecutor
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)